docs/API_documentation/Overview/Getting_access.md

```diff
@@ -16,15 +16,14 @@ The `POST` request contains the following parameters:
 - *password* - your password
 
 !!! Note
-    SSO users (usually business users) need to have an IAM service account. Contact the platform administrator to have this account created. 
+    SSO users (usually business users) need to have an Identity Access Management (IAM) service account. Contact the platform administrator to have this account created. 
 
 !!! Note
     In some installations the users obtain access token through exchange token procedure. Contact the platform administrator for more information.
 
 **How to get access token**
 
 You can obtain a REST API access token by sending the `POST` request on an authentication endpoint:
-
 ```
 curl -X - `POST` \                                                                                     
```

Comment:
I would add "Enter the command line interface on your machine and run the following command:"